Anlab.Mvc/Views/Shared/_OrderDetails.cshtml

```diff
@@ -121,9 +121,9 @@
                 }
             </td>
             <td>
-                @if(Model.HideLabDetails)
+                @if (Model.HideLabDetails)
                 {
-                    <a href="/analysis/@temp.Category/@temp.Sop" target="_blank">@temp.Sop</a>
+                    <a asp-controller="Analysis" asp-action="@temp.Category" asp-route-id="@temp.Sop" target="_blank">@temp.Sop</a>

```

Comment:
@laholstege You just need:
<a asp-controller="Analysis" asp-action="Details" asp-route-id="@temp.Sop" target="_blank">@temp.Sop</a>

Reply:
@jSylvestre I wanted it to be /analysis/soil/123 instead of /analysis/details/123. will this not work?

Reply:
@laholstege It should but that route is really only needed from React. It makes it more difficult to maintain.